python/Ganga/Runtime/bootstrap.py

```diff
@@ -762,15 +762,15 @@ def initEnvironment(opt_rexec):
 
             
             import inspect
+            from os.path import expandvars, expanduser
+
             GangaRootPath = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))), '../..'))
             def transform(x):
-                return os.path.normpath(Ganga.Utility.files.expandfilename(os.path.join(GangaRootPath,x)))
-
-            from os.path import expandvars, expanduser
+                return os.path.normpath(Ganga.Utility.files.expandfilename(
+                        os.path.join(GangaRootPath, expandvars(x))))
 
             paths = map(transform, filter(lambda x: expandvars(expanduser(x)), config['RUNTIME_PATH'].split(':')))
```

Comment:
@mjmottram This line looks like it's trying to do an expandvars/expanduser does this map not perform as expected?